src/themes/masschallenge/collections/message.variables

```diff
@@ -0,0 +1,6 @@
+@dangerBorderColor: #C7BA9E;
+@successBorderColor: #B2D4DC;
+@dangerBackground: #FEFAF4;
+@successBackground: #F9FFFF;
```

Comment:
elsewhere hexadecimal codes are written in lowercase. I have a mild preference for lowercase, but mostly we should be consistent, so either all lower or all uppercase.